### DISCUSSION TITLE
Next Update Release?

### DISCUSSION THREAD

**Discussion by bostonmyk** [PygmalionOfCyprus/cmo-db-requests#1228] [General]
Can 494 be released as part of beta?

**Comment by PygmalionOfCyprus** [comment]
We've passed it on to the other devs; AFAIK they're just adding some sim-side code to support new features and then you guys will have it! 🙂

**Comment by bostonmyk** [comment]
Awesome. Thanks!

**Comment by PygmalionOfCyprus** [comment]
Initial release w/ private [Build 1254](https://www.matrixgames.com/forums/viewtopic.php?f=11659&t=384771).

**Comment by bostonmyk** [comment]
Not in public builds yet. Up to you my man, but would gladly test stuff if they ended up in public beta.

**Reply by PygmalionOfCyprus** [reply]
It's not up to me. 🥲 Not entirely, anyway.

The prevailing opinion amongst the team is that we want to hold off on a public beta release until support for all the new features is implemented sim-side. I think that's the right call, as it'd otherwise be a fairly frustrating release with lots of "We added this, but you can't use it yet." One can't exactly test that which isn't there!

We were intending to have all this ready much earlier but we got schwacked with some surprises. Currently, it's likely to go public by the end of the month.

**Comment by bostonmyk** [comment]
Ok thanks.

**Comment by KLABDB3K** [comment]
Good to know. Thanks.

**Comment by bostonmyk** [comment]
Just got it in today's update. Thank you!
